File: 11530/Main.java
import java.io.*;

public class Main {
   final static int[] pressTimes = {
      1, 2, 3,
      1, 2, 3,
      1, 2, 3,
      1, 2, 3,
      1, 2, 3,
      1, 2, 3, 4,
      1, 2, 3,
      1, 2, 3, 4
   };
   public static void main(String args[]) throws IOException {
      String answer = "";
      BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
      BufferedWriter out = new BufferedWriter(new OutputStreamWriter(System.out));

      int n = Integer.parseInt(in.readLine());
      for (int testCase = 1; testCase <= n; testCase++) {
         int count = 0;
         String line = in.readLine();
         for (int i = 0; i < line.length(); i++) {
            char c = line.charAt(i);
            if (c == ' ') {
               ++count;
            } else {
               count += pressTimes[c - 'a'];
            }
         }
         answer += ("Case #" + Integer.toString(testCase) + ": " + Integer.toString(count) + "\n");
      }
      out.write(answer);
      out.flush();
   }
}

File: 11530/main.c
#include <stdio.h>
#include <stdlib.h>

const static int press_times[26] = {
   1, 2, 3,
   1, 2, 3,
   1, 2, 3,
   1, 2, 3,
   1, 2, 3,
   1, 2, 3, 4,
   1, 2, 3,
   1, 2, 3, 4
};

int main(int argc, char* argv[])
{
   char input[101];
   int i, n, count, test_case = 1;

   gets(input);
   n = atoi(input);
   for (test_case = 1; test_case <= n; test_case++) {
      gets(input);
      count = 0;
      for (i = 0; input[i] != '\0'; i++) {
         if (input[i] == ' ') {
            ++count;
         } else {
            count += press_times[input[i] - 'a'];
         }
      }
      printf("Case #%d: %d\n", test_case, count);
   }

   return 0;
}
